src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+      Hello, world!
+  </body>
```

Comment:
```suggestion
  <body>
    Hello, world!
  </body>
```
Pay attention to formatting. Too many indentations here. There should be 2 instead of 4.
You may read about it here https://mate-academy.github.io/style-guides/htmlcss.html